src/webconnector/package-lock.json

```diff
@@ -2264,7 +2264,7 @@
         "browserify-rsa": "^4.0.0",
         "create-hash": "^1.1.0",
         "create-hmac": "^1.1.2",
-        "elliptic": "^6.0.0",
```

Comment:
Heb je deze bewust ge-update? Kan vast geen kwaad maar als het per ongeluk was kan je hem beter weglaten. Sowieso is het iets netter om het dan in twee commits te doen waarbij je in de commit die deze update aan kan geven wat de reden daarvoor was.

(niet belangrijk genoeg om nog aan te passen hier; maar voor een volgende keer)

Reply:
Ja bewust gedaan, wanneer je de repro opende gaf Git een security warning met de suggestie om deze te updaten. Ja eens, beter om dit voortaan als een aparte PR te verwerken